ops/terraform/modules/resources/bfd_cw_dashboards/main.tf

```diff
@@ -0,0 +1,6 @@
+resource "aws_cloudwatch_dashboard" "bfd-dashboards" {
+  dashboard_name = var.dashboard_name
+  dashboard_body = templatefile("../../../modules/resources/bfd_cw_dashboards/templates/bfd-dashboards.json", {
+    dashboard_namespace = var.dashboard_namespace
+  })
+}
```

Comment:
This isn't a blocker, but could you fix your editor to avoid these?

Reply:
> Thanks for the contribution!
> 
> Could you take a look at some of the suggestion? We ought to avoid introducing more variability to the environments with a new module. Let's leverage the environment name from `var.env_config.env` instead.

@mjburling Changes have been made to the branch as requested,